### PR TITLE
logging and plugins

### PR DESCRIPTION
**Pull request by sosiouxme**

### CODE REVIEW

**Comment by sosiouxme** [comment]
@jcantrill @ewolinetz PTAL

File: elasticsearch/logging.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ logger:
   action: WARN
   # reduce the logging for aws, too much is logged under the default INFO
   com.amazonaws: WARN
-  #io.fabric8.elasticsearch: DEBUG
-  #io.fabric8.kubernetes: DEBUG
-  #io.fabric8.elasticsearch.plugin.acl: DEBUG
+  io.fabric8.elasticsearch: ${PLUGIN_LOGLEVEL}
```
Comment (ewolinetz) [inline elasticsearch/logging.yml]: Setting the log level for this package should also set the log level for io.fabric8.elasticsearch.plugin.acl; we should either let you set them separately, or remove line 11

**Comment by ewolinetz** [comment]
one nit otherwise LGTM

File: elasticsearch/logging.yml (hunk 1 above)
Comment (sosiouxme) [inline elasticsearch/logging.yml]: Thanks. Durrrr :)

**Comment by sosiouxme** [comment]
I added on an improvement to the RAM allocation calculator. Now it won't bork on 1G or under-allocate odd Gigs.

**Comment by ewolinetz** [comment]
LGTM

**Pull request merged**